src/runtime_src/core/pcie/windows/system_windows.cpp

```diff
@@ -174,7 +186,7 @@ get_mgmtpf_device(device::id_type id) const
 
 void
 system_windows::
-program_plp(std::shared_ptr<device> dev, std::vector<char> buffer) const 
+program_plp(std::shared_ptr<device> dev, std::vector<char> buffer) const
```

Comment:
Not part of this PR, but still has to be fixed.  Do not pass by value, especially `buffer ` looks very inefficient, but also dev should be pass by const reference.

Reply:
I wonder how this crept in. I went ahead and updated the performance critical std::vector reference change `system::program_plp(std::shared_ptr<device> dev, const std::vector<char> &buffer)` 
Leaving out the shared_ptr device change for future.